{"level":"info","message":"Server running on port 5000","timestamp":"2024-09-07T18:29:56.838Z"}
{"level":"error","message":"Error connecting to database: getaddrinfo ENOTFOUND dpg-craklelds78s73d5o7u0-a","name":"SequelizeHostNotFoundError","original":{"code":"ENOTFOUND","errno":-3008,"hostname":"dpg-craklelds78s73d5o7u0-a","syscall":"getaddrinfo"},"parent":{"code":"ENOTFOUND","errno":-3008,"hostname":"dpg-craklelds78s73d5o7u0-a","syscall":"getaddrinfo"},"stack":"SequelizeHostNotFoundError: getaddrinfo ENOTFOUND dpg-craklelds78s73d5o7u0-a\n    at Client._connectionCallback (/home/runner/talkie-backend/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:136:24)\n    at Client._handleErrorWhileConnecting (/home/runner/talkie-backend/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/runner/talkie-backend/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:519:28)\n    at Socket.reportStreamError (/home/runner/talkie-backend/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:519:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)","timestamp":"2024-09-07T18:29:58.263Z"}
{"level":"info","message":"Server running on port 5000","timestamp":"2024-09-07T18:34:49.915Z"}
{"level":"error","message":"Error connecting to database: SSL/TLS required","name":"SequelizeConnectionError","original":{"code":"28000","length":37,"name":"error","severity":"FATAL"},"parent":{"code":"28000","length":37,"name":"error","severity":"FATAL"},"stack":"SequelizeConnectionError: SSL/TLS required\n    at Client._connectionCallback (/home/runner/talkie-backend/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/runner/talkie-backend/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorMessage (/home/runner/talkie-backend/node_modules/pg/lib/client.js:347:19)\n    at Connection.emit (node:events:519:28)\n    at /home/runner/talkie-backend/node_modules/pg/lib/connection.js:117:12\n    at Parser.parse (/home/runner/talkie-backend/node_modules/pg-protocol/dist/parser.js:36:17)\n    at Socket.<anonymous> (/home/runner/talkie-backend/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)","timestamp":"2024-09-07T18:34:51.566Z"}
{"level":"info","message":"Server running on port 5000","timestamp":"2024-09-07T18:36:18.899Z"}
{"level":"info","message":"Connected to PostgreSQL database","timestamp":"2024-09-07T18:36:25.535Z"}
{"level":"info","message":"Server running on port 5000","timestamp":"2024-09-08T13:13:45.901Z"}
{"level":"info","message":"Connected to PostgreSQL database","timestamp":"2024-09-08T13:13:51.723Z"}
{"level":"info","message":"Server running on port 5000","timestamp":"2024-09-08T14:10:37.644Z"}
{"level":"info","message":"Connected to PostgreSQL database","timestamp":"2024-09-08T14:10:42.838Z"}
{"level":"info","message":"Server running on port 5000","timestamp":"2024-09-08T14:47:10.373Z"}
{"level":"info","message":"Connected to PostgreSQL database","timestamp":"2024-09-08T14:47:16.313Z"}
{"level":"info","message":"Server running on port 5000","timestamp":"2024-09-08T14:53:35.130Z"}
{"level":"info","message":"Connected to PostgreSQL database","timestamp":"2024-09-08T14:53:43.651Z"}
{"code":"42P01","file":"parse_relation.c","length":108,"level":"error","line":"1449","message":"Error fetching user profile: relation \"accounts\" does not exist","name":"error","position":"402","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"accounts\" does not exist\n    at /home/runner/talkie-backend/node_modules/pg-pool/index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async exports.getUserProfile (/home/runner/talkie-backend/src/controllers/userController.js:41:20)","timestamp":"2024-09-08T15:09:25.053Z"}
{"code":"42P01","file":"parse_relation.c","length":108,"level":"error","line":"1449","message":"Error fetching user profile: relation \"accounts\" does not exist","name":"error","position":"402","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"accounts\" does not exist\n    at /home/runner/talkie-backend/node_modules/pg-pool/index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async exports.getUserProfile (/home/runner/talkie-backend/src/controllers/userController.js:41:20)","timestamp":"2024-09-08T15:09:27.995Z"}
{"code":"42P01","file":"parse_relation.c","length":108,"level":"error","line":"1449","message":"Error fetching user profile: relation \"accounts\" does not exist","name":"error","position":"402","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"accounts\" does not exist\n    at /home/runner/talkie-backend/node_modules/pg-pool/index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async exports.getUserProfile (/home/runner/talkie-backend/src/controllers/userController.js:41:20)","timestamp":"2024-09-08T15:10:00.436Z"}
{"code":"42P01","file":"parse_relation.c","length":108,"level":"error","line":"1449","message":"Error fetching user profile: relation \"accounts\" does not exist","name":"error","position":"402","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"accounts\" does not exist\n    at /home/runner/talkie-backend/node_modules/pg-pool/index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async exports.getUserProfile (/home/runner/talkie-backend/src/controllers/userController.js:41:20)","timestamp":"2024-09-08T15:10:03.304Z"}
{"code":"42P01","file":"parse_relation.c","length":108,"level":"error","line":"1449","message":"Error fetching user profile: relation \"accounts\" does not exist","name":"error","position":"402","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"accounts\" does not exist\n    at /home/runner/talkie-backend/node_modules/pg-pool/index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async exports.getUserProfile (/home/runner/talkie-backend/src/controllers/userController.js:41:20)","timestamp":"2024-09-08T15:12:26.622Z"}
{"code":"42P01","file":"parse_relation.c","length":108,"level":"error","line":"1449","message":"Error fetching user profile: relation \"accounts\" does not exist","name":"error","position":"402","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"accounts\" does not exist\n    at /home/runner/talkie-backend/node_modules/pg-pool/index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async exports.getUserProfile (/home/runner/talkie-backend/src/controllers/userController.js:41:20)","timestamp":"2024-09-08T15:12:31.189Z"}
{"code":"42P01","file":"parse_relation.c","length":108,"level":"error","line":"1449","message":"Error fetching user profile: relation \"accounts\" does not exist","name":"error","position":"402","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"accounts\" does not exist\n    at /home/runner/talkie-backend/node_modules/pg-pool/index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async exports.getUserProfile (/home/runner/talkie-backend/src/controllers/userController.js:41:20)","timestamp":"2024-09-08T15:13:36.753Z"}
{"level":"info","message":"Server running on port 5000","timestamp":"2024-09-08T15:20:33.496Z"}
{"level":"info","message":"Connected to PostgreSQL database","timestamp":"2024-09-08T15:20:38.404Z"}
{"level":"info","message":"Server running on port 5000","timestamp":"2024-09-08T16:04:51.825Z"}
{"level":"error","message":"Error connecting to database: Connection terminated unexpectedly","stack":"Error: Connection terminated unexpectedly\n    at /home/runner/talkie-backend/node_modules/pg-pool/index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async connectDB (/home/runner/talkie-backend/src/config/database.js:25:5)","timestamp":"2024-09-08T16:07:07.854Z"}
{"level":"info","message":"Server running on port 5000","timestamp":"2024-09-08T16:11:28.027Z"}
{"level":"info","message":"Connected to PostgreSQL database","timestamp":"2024-09-08T16:11:34.704Z"}
